PLEP-0007.rst

```diff
@@ -0,0 +1,266 @@
+| :access: ``plasmapy.plasma``                                                 |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A collection of functions, classes, and sub-packages that fully defines a  |
+|   plasma.  This would include the plasma's species constituents and physical |
+|   parameters (like temperature, boundary conditions, magnetic fields, etc.). |
+|                                                                              |
+|   Any tools that go into defining a plasma or its environment (e.g. a field  |
+|   solver) should be included in a sub-package withing ``plasmapy.plasma``.   |
++------------------------------------------------------------------------------+
+| .. _simulation:                                                              |
+|                                                                              |
+| **simulation**                                                               |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/simulation/``                                       |
+| :access: ``plasmapy.simulation``                                             |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A collection of functions, classes, and sub-packages focused on            |
+|   interfacing with simulations and/or running simulations.                   |
+|                                                                              |
+|   If a new feature falls under the scope of the ``analysis`` and/or          |
+|   ``diagnostics`` sub-packages, then the feature should be included one of   |
+|   respective packages.  For example, a synthetic diagnostic should be        |
+|   included in the ``plasmapy.diagnostics`` sub-package.                      |
++------------------------------------------------------------------------------+
+| .. _tests:                                                                   |
+|                                                                              |
+| **tests**                                                                    |
++------------------------------------------------------------------------------+
+| :directory: ``./plasmapy/tests/``                                            |
+| :access: ``plasmapy.tests``                                                  |
+| :PLEP:                                                                       |
+| :scope:                                                                      |
+|   A collection of tests for top-level modules (i.e. functions and classes    |
+|   defined in top-level ``.py`` files).                                       |
```

Comment:
We'll need to expand this so that `plasmapy.tests` includes test helper functionality too (akin to `astropy.tests`).